runners/core-java/pom.xml

```diff
@@ -126,11 +126,11 @@
                      the second relocation. -->
                 <relocation>
                   <pattern>com.google.common</pattern>
-                  <shadedPattern>org.apache.beam.sdk.repackaged.com.google.common</shadedPattern>
+                  <shadedPattern>${project.groupId}.${project.artifactId}.repackaged.com.google.common</shadedPattern>
```

Comment:
hyphen is not a valid package name character so what is actually generated here?